src/config/example.toml

```diff
@@ -41,3 +39,15 @@ stream_enable_executor_row_count = false
 stream_connector_message_buffer_size = 16
 unsafe_stream_extreme_cache_size = 1024
 stream_chunk_size = 1024
+
+[meta]
+listen_addr = "127.0.0.1:5690"
+
+[frontend]
+listen_addr = "127.0.0.1:4566"
+
+[compute_node]
```

Comment:
How about `[compute]`? Only for consistency of style 🤣

Reply:
Or change others to `[frontend_node]`, `[compactor_node]`...? 🤪

Reply:
This looks misleading. What does it mean if we have multiple frontends or compute nodes?

Reply:
They can read from different files?

Reply:
If you're suggesting different `toml`s for different physical nodes, then we should also naturally make `toml` node-type-specific. That is, if we should have to specify different config for **one** frontend, why don't we specify different config for all frontends, where other sections like `streaming` will be redundant?

Reply:
@fuyufjh mentioned in https://github.com/risingwavelabs/rfcs/pull/34#issuecomment-1381458145 that 

> Multiple files may be troublesome for deployment.

So I guess the ideal way is to use the CLI?

Reply:
Yes. I think so. 🥵 Maybe we really need some CLI-only configuration items.

Reply:
Though making those items configurable from the file won't hurt anyone, right? If one needs multiple nodes of the same type or simply prefers CLI, they can always use it. Adding them to the file just gives the user another choice?

Reply:
Still confusing to me. 🥺 Users may think that the "frontend address" must be the same for the one in the single and shared toml, with the one overridden in frontend CLI, though we know that the address in toml is never read by other components in the cluster. The essential reason is that, this is not a cluster-level config.

Reply:
Either looks good to me.

> single and shared toml

This might not be an enforced rule in my mind. **In principle**, the process-level config *should be* different for each node, because there are local files and nothing enforces them to be identical.

What we did is to make these users happy, for example, by allowing the node-level arguments configurable via CLI args, so that they can use a shared config file and different CLI args. But this is only a sugar.

Saying that there is another group of users, who would like to use a toml library to generate a full config for each node respectively, I think this is also fine. Actually, MySQL/PG users have to do thing, considering that MySQL/PG can be clustered to be Paxos HA group and them must listen on different IPs.

Reply:
I prefer single toml because there has to be some configs in the toml file but not in CLI that can vary from nodes.

Reply:
`toml`s under `config` directory are used by RiseDev, which does not follow the way of "another group of users". So what about just leaving the addresses fields unset, as we always specify the addresses with CLI args? Using a comprehensive config file is also acceptable from this PR, but it's not a way **we**, as RiseDev users, to use it.

Reply:
> what about just leaving the addresses fields unset, as we always specify the addresses with CLI args

Since using multiple files won't be a problem, I think these fields can still appear in the file 👀 

> we should also naturally make toml node-type-specific. That is, if we should have to specify different config for every single frontend, why don't we specify different config for all frontends, where other sections like streaming will be redundant?

We can certainly consider different schema for each node type. But let's not introduce too much change in this PR.